src/main/java/me/EvsDev/EnderDragonTweaks/ConfigManager.java

```diff
@@ -46,13 +49,15 @@ public String getString(String name) {
 
 	public String formatDefeatAnnouncementMessage(Player killer, World theEnd) {
 		String message = ChatColor.translateAlternateColorCodes('&', this.getString(entry_defeatMessage));
+                int playerRadius = getInt(entry_playerDistanceFromOrigin);
```

Comment:
Funky indent.

Reply:
I'll make sure to set my IDE to use the correct delimiter